Tools/build/deepfreeze.py

```diff
@@ -17,13 +17,13 @@
 from typing import Dict, FrozenSet, TextIO, Tuple
 
 import umarshal
+import opcode_for_build as opcode
```

Comment:
This isn't a normal import so it shouldn't look like one. Maybe something like `opcode = opcode_finder.get_opcodes()`